src/triggers/CBLlot.trigger

```diff
@@ -0,0 +1,18 @@
+trigger CBLlot on CBL_Lot__C(after insert){
+  for (CBL_Lot__C newLot : Trigger.new) {
+    if (newLot.CBL2_Lot__c <> null) {
+      for(CBL2_Lot__c cbl2lot : [SELECT lotArea__c, lotName__c, lotAreaUnits__c, lotCode__c, Lot_Code_Guat__c, lotCoffeeVariety__c, Lot_Latitude__c, Lot_Longitude__c, lotNumberOfPlants__c, lotPlantingDate__c, Submission__c FROM CBL2_Lot__c WHERE Id = :newLot.CBL2_Lot__c]){ 
```

Comment:
By batch safe, Steve means that if there are more than 100 items in Trigger.new, the trigger will fail.

How about using an "IN" statement and double nested for statements? That'll make it more batch-safe.

```apex
lot_ids = new List<Id>();
for (CBL_Lot__C newLot : Trigger.new) {
       lot_ids.add(newLot.id);
}

for(CBL2_Lot__c[] cbl2lots : [SELECT lotArea__c, lotName__c, lotAreaUnits__c, lotCode__c, Lot_Code_Guat__c, lotCoffeeVariety__c, Lot_Latitude__c, Lot_Longitude__c, lotNumberOfPlants__c, lotPlantingDate__c, Submission__c FROM CBL2_Lot__c WHERE Id IN :lot_ids]){
       for (CBL2_Lot__c cbl2lot: cbl2lots) {
              // Do what you want with the lots;
              ...
       }
} 
```